config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/gcp-compute-persistent-disk-csi-driver-postsubmits.yaml

```diff
@@ -24,4 +24,30 @@ periodics:
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
+- name: ci-gcp-compute-persistent-disk-csi-driver-k8s-integration-stable-1-13-4
+  interval: 4h
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-master
+      args:
+      - "--repo=sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
+      - "--root=/go/src"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--clean"
+      - "--timeout=60" # Minutes
+      - "--scenario=execute"
+      - "--" # end bootstrap args, scenario args below
+      - "test/run-k8s-integration.sh"
+      - "--kube-version=1.13.4"
```

Comment:
Actually, does kubetest already support finding the latest official 1.13 tag?  I know there's already tags like k8sstable and such. Is there one per release?

Reply:
we currently manually download the release on our end so even if kubetest does support it this would be extra work to migrate over to that which we can do later